src/System.Console/src/System/Console.cs

```diff
@@ -326,7 +327,7 @@ public static void SetCursorPosition(int left, int top)
             ConsolePal.SetCursorPosition(left, top);
         }
 
-        public static event ConsoleCancelEventHandler CancelKeyPress
+        public static event ConsoleCancelEventHandler CancelKeyPress // TODO: Ask for review
```

Comment:
This PR is ready for review

@safern @buyaa-n could you also please take a closer look at this code block?

Reply:
Events should be nullable: https://github.com/dotnet/coreclr/pull/25752